collector/src/main/java/com/navercorp/pinpoint/collector/receiver/udp/UDPReceiver.java

```diff
@@ -221,14 +216,9 @@ private void bindSocket(DatagramSocket socket, InetSocketAddress bindAddress) {
     }
 
     private int getPacketPoolSize() {
-        int threadSize = worker.getThreadSize();
-        int queueSize = worker.getQueueSize();
-
-        return threadSize + queueSize + ioThreadSize;
+        return 1024*5;
```

Comment:
Is it fixed value?

Reply:
This is a temporary code
In the next issue, I will extract the datagram pool into a Spring-Bean

Reply:
good :)